src/Text/Pandoc/Writers/Ms.hs

```diff
@@ -351,6 +351,9 @@ inlineToMs :: PandocMonad m => WriterOptions -> Inline -> MS m (Doc Text)
 inlineToMs opts (Span _ ils) = inlineListToMs opts ils
 inlineToMs opts (Emph lst) =
   withFontFeature 'I' (inlineListToMs opts lst)
+inlineToMs opts (Underline lst) = do
+  contents <- inlineListToMs opts lst
+  return $ literal ".UL " <> contents <> " "
```

Comment:
This isn't what you want here.  Note that the `.UL` can only go at the beginning of a line, and this might be rendered anywhere.  If it's not possible to use `withFontFeature` as we do for Emph, then at the very least we need to force a `cr` before and after here.  But that could cause problems when the inline sequence is itself the argument of something like `.SE`.  Maybe the best approach would just be treating underline exactly like emphasis in this format?

Reply:
Updated to behave the same as `Emph`.